packages/connect-react/src/components/ConfigureFilePicker/index.tsx

```
@@ -38,12 +40,58 @@ export interface FilePickerIcons {
   file?: ReactNode;
 }
 
+/**
+ * Simple SVG icon components
+ */
+const FolderIcon = () => (
+  <svg width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" strokeWidth="1.5" strokeLinecap="round" strokeLinejoin="round">
```

Biome (2.3.14)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)